backend/src/api/routes/userRoute.ts

```diff
@@ -1,11 +1,12 @@
 import { Router, Request } from 'express';
 import { run } from '../../common/utils/routeHelper';
-import { getUsers, getUserById } from '../../services/userService';
+import { getUsers, getUserById, editProfile } from '../../services/userService';
 
 const router = Router();
 
 router
   .get('/', run(getUsers))
-  .get('/:id', run((req: Request) => getUserById(req.params.id)));
+  .get('/:id', run((req: Request) => getUserById(req.params.id)))
+  .put('/', run((req: Request) => editProfile(req.body)));
```

Comment:
As far as I know, according to the REST, id should be passed as query param, but you have it in req.body
```suggestion
  .put('/:id', run((req: Request) => editProfile(req.body)));
```

Reply:
I made a request that was specified in the task